mlflow/__init__.py

```diff
@@ -221,6 +222,7 @@
     "set_tag",
     "set_tags",
     "set_tracking_uri",
+    "shut_down_async_logging",
```

Comment:
```suggestion
    "_shut_down_async_logging",
```

I don't think we need to expose this as a public API.